src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/CompilerScope.Storage.cs

```diff
@@ -47,7 +47,9 @@ internal LocalStorage(LambdaCompiler compiler, ParameterExpression variable)
                 // the compiler when emitting an inlined lambda invoke, to
                 // handle ByRef parameters. BlockExpression prevents this
                 // from being exposed to user created trees.
-                _local = compiler.GetNamedLocal(variable.IsByRef ? variable.Type.MakeByRefType() : variable.Type, variable);
+
+                // Set name if DebugInfoGenerator support is brought back.
```

Comment:
Is there any impediment to actually bringing debug info generation back? I've never used the feature myself, but I can certainly see how one might want to.